tests/js/basis/basis.js

```diff
@@ -218,4 +218,14 @@ describe("Basis", () => {
         basis.translateByVector(translationVector);
         assertDeepAlmostEqual(basis.coordinates, C2H4Translated.basis.coordinates);
     });
+
+    it("should strip label and return atomic symbol", () => {
```

Comment:
Should we also test for the opposite - getting the label itself from the elementWithLabel string?

Reply:
@timurbazhirov added a separate test below, you may have another look.